### DISCUSSION TITLE
"Table of Contents" with "Sidebar"

### DISCUSSION THREAD

**Discussion by dylan-tock** [community/community#36192] [Other Feature Feedback, Questions, & Ideas]
Is there a way to combine the [auto generated Table of Contents](https://github.blog/changelog/2021-08-19-table-of-content-for-wikis/) with contents of a [`_Sidebar.md`](https://docs.github.com/en/communities/documenting-your-project-with-wikis/creating-a-footer-or-sidebar-for-your-wiki) file?  The Wiki I'm working in has a pre-existing Sidebar and, while I'm ambivalent, others like it. I really want Table of Contents for my pages. Is there a way to get the ToC without removing the _Sidebar file?

**Comment by igorsantos07** [comment]
I'm not sure this request is still valid. Right now, we have a wiki with both the custom sidebar and the (closed, ugly, mostly useless) ToC above it.

![image](https://github.com/user-attachments/assets/0c21f470-b6c5-4454-a770-aa71d95f9534)
